src/CSS.php

```diff
@@ -66,6 +66,29 @@ public function setImportExtensions(array $extensions)
     }
 
     /**
+     * Move any import statements to the top
+     *
+     * @param $content string Nearly finished CSS content
+     *
+     * @return string
+     */
+    protected function moveImportsToTop($content) {
+
+        if (preg_match_all('#@import[^;]+;#', $content, $matches)) {
+
+            // remove from content
+            foreach($matches[0] as $import) {
```

Comment:
According to PSR-2 coding standard, there should be a space between `foreach` and `(`